swarm/network/stream/stream.go

```diff
@@ -259,9 +241,40 @@ func NewRegistry(localID enode.ID, delivery *Delivery, syncChunkStore chunk.Fetc
 		}()
 	}
 
+	go streamer.emitPeriodicSubscriptionMetrics()
+
 	return streamer
 }
 
+func (r *Registry) emitPeriodicSubscriptionMetrics() {
+	for {
+		select {
+		case <-time.After(10 * time.Second):
```

Comment:
This function should return on <-r.quit to terminate the goroutine in which it runs.

Reply:
done